src/models.py

```diff
@@ -63,6 +63,7 @@ class Course(db.Model):
     securityCode = db.Column(db.Integer)
     isArchived = db.Column(db.Boolean, default=False)
     teacher_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    secondaryTeachers = db.Column(db.String(255),default=",")
```

Comment:
dont default to ","

I wrote about this more in depth in my other comments (with sample code)